appeals/web/src/server/lib/mappers/appeal-details/s78.mapper.js

```diff
@@ -0,0 +1,217 @@
+import config from '#environment/config.js';
```

Comment:
Just a note on the file name, the appellant case equivalent is using the prefix `fpa` rather than `s78`, for consistency this file should probably be renamed `fpa.mapper.js`

Reply:
gotcha